src/css/admin-settings.scss

```diff
@@ -1,49 +1,85 @@
-:root {
+@import "../content-helper/common/css/variables";
+@import "../content-helper/common/css/functions";
+
+.settings_page_parsely {
 	--padding-default: 15px;
-}
 
-#track-post-types {
-	max-width: 550px;
-	width: 100%;
-}
+	#track-post-types {
+		max-width: 550px;
+		width: 100%;
 
-#track-post-types th,
-#track-post-types td {
-	display: table-cell;
-	padding: var(--padding-default);
-	text-align: center;
-	vertical-align: middle;
-}
+		th,
+		td {
+			display: table-cell;
+			padding: var(--padding-default);
+			text-align: center;
+			vertical-align: middle;
+		}
 
-#track-post-types td {
-	padding: 0; /* Set padding to 0 so labels can take the space */
-}
+		/* Set padding to 0 so labels can take all the space */
+		td {
+			padding: 0;
+		}
 
-/* Make radio box clickable in whole cell */
-#track-post-types td label {
-	display: inline-block;
-	margin: 0 !important;
-	padding-top: var(--padding-default);
-	padding-bottom: var(--padding-default);
-	width: 100%;
-}
+		/* Make radio box clickable in whole cell */
+		td label {
+			display: inline-block;
+			margin: 0 !important;
+			padding-top: var(--padding-default);
+			padding-bottom: var(--padding-default);
+			width: 100%;
+		}
 
-#track-post-types thead th:nth-child(3) {
-	word-break: break-word;
-}
+		thead th:nth-child(3) {
+			word-break: break-word;
+		}
+	}
 
-@media only screen and (max-width: 380px) {
+	fieldset:disabled,
```

Comment:
`fieldset` and badge styles have been added in this section, until `@media` styles.